packages/client/hmi-client/src/components/data-explorer/search-item.vue

```diff
@@ -120,17 +142,20 @@ const formatFeatures = () => {
 			<footer><!--pill tags if already in another project--></footer>
 		</div>
 		<div class="right">
-			<figure v-if="resourceType === ResourceType.XDD && extractionsWithImages.length > 0">
+			<figure v-if="resourceType === ResourceType.XDD && asset.relatedExtractions">
 				<img
 					v-if="relatedAsset && relatedAsset.properties.image"
 					:src="`data:image/jpeg;base64,${relatedAsset.properties.image}`"
 					class="extracted-assets"
 					alt="asset"
 				/>
 				<div class="asset-nav-arrows">
-					<i class="pi pi-arrow-left" @click="paginationMovement(-1)"></i>
-					Asset {{ relatedAssetPage + 1 }} of {{ extractionsWithImages.length }}
-					<i class="pi pi-arrow-right" @click="paginationMovement(1)"></i>
+					<i class="pi pi-arrow-left" @click.stop="paginationMovement(-1)"></i>
+					<template v-if="extractionsWithImages.length > 0">
+						Asset {{ relatedAssetPage + 1 }} of {{ extractionsWithImages.length }}
+					</template>
+					<template v-else> No {{ chosenExtractionFilter }}s </template>
+					<i class="pi pi-arrow-right" @click.stop="paginationMovement(1)"></i>
```

Comment:
Nitpick: I would recommend we don't use the pagination nomenclature here, maybe `preview` is a better term?

Tabular data will typically not show up well in small sizes, and to some extent equations because of with/height ratios ... I wonder if we need to go back to Jamie for this for more thoughts.

Reply:
Similar situation as above, they are undefined anyways so I'll apply proper filters once we have chosen which ones/once the backend works